guide/payments/privacy.md

```diff
@@ -0,0 +1,143 @@
+> * The person who sold you the bitcoins follows their trail on the blockchain and finds your paper wallet containing $1 million.
+> * He mentions it to someone in a cafe or bar.
+> * Word gets around. A burglar raids your home. Kidnappers capture your children and know exactly how much to demand in ransom.
+
+-->
+
+
+{% include fact/dos.html %}
+- Show a new address when the receive screen is entered.
+- Make it easy to generate as many addresses as the receiver needs.
+- Before broadcasting a transaction, inform senders if an address is already used.
+{% include fact/close.html %}
+
+{% include fact/donts.html %}
+- Make it easy to reuse an address.
+{% include fact/close.html %}
+
+### Keep track of who knows about an address
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/address-labeling.jpg"
+   retina = "/assets/images/guide/payments/privacy/address-labeling@2x.jpg"
+   alt-text = "A list of addresses with tags. Two are red, two are purple and one is blue"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+When [receiving bitcoin]({{ "/guide/payments/receive/" | relative_url }}) we can [add additional details to a payment]({{ "/guide/payments/receive/#inputting-additional-payment-details" | relative_url }}). Not only does this help to contextualize payments, but it also enables preventative measures for preserving privacy. Labeling addresses with the counterparty's name involved in each transaction can inform decisions for which [UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }}) are selected in future transaction inputs.
+
+To make this UTXO selection easier, some interfaces make it possible to select/filter UTXOs that cluster under one label.
+
+_Check out the glossary to learn more about [how coin control works]({{ "/guide/glossary/#coin-control" | relative_url }})._
+
+</div>
+
+### Increase anonymity by collaborating with others
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/coin-join.jpg"
+   retina = "/assets/images/guide/payments/privacy/coin-join@2x.jpg"
+   alt-text = "Example CoinJoin transaction with 3 participants with 1 input each, which get broken up into 5 equal outputs."
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+The "common input ownership" heuristic[^4] assumes in a transaction, it is likely that all inputs belong to the same owner. [CoinJoins]({{ "/guide/glossary/#coinjoin" | relative_url }}) is a technique where multiple participants collaborate on a transaction to break this heuristic. It becomes more difficult (although not impossible) to determine the individuals who own the outputs are since all the outputs typically have the same denomination. With CoinJoins, as with any other anonymity network, a large and diverse group of participants provides the best protection.
```

Comment:
```suggestion
The "common input ownership" heuristic[^4] assumes that all inputs in a transaction likely belong to the same owner. [CoinJoins]({{ "/guide/glossary/#coinjoin" | relative_url }}) is a technique where multiple participants collaborate on a transaction to break this heuristic. It becomes more difficult (although not impossible) to determine the individuals who own the outputs are since all the outputs typically have the same denomination. With CoinJoins, as with any other anonymity network, a large and diverse group of participants provides the best protection.
```